src/docs/ganache/reference/cli-options.md

```diff
@@ -0,0 +1,917 @@
+If you need to use an Infura API key secret, you would use it as follows:
+
+```bash
+wss://:<API-KEY-SECRET>@mainnet.infura.com/ws/v3/<API-KEY>
+```
+
+Alternatively, you can use the `--fork.username` and `--fork.password` options.
+
+**`--fork.network`**
+
+=== "Syntax"
+    ```bash
+    --fork.network=<STRING>
+    ```
+
+=== "Example"
+
+    ```bash
+    --fork.network="goerli"
+    ```
+
+The network to fork. Valid options are `mainnet`, `ropsten`, `kovan`, `rinkeby`, `goerli`, `görli`,
+and `sepolia`.
+
+Use the command `ganache --fork` to automatically fork Mainnet at the latest block.
+
+**`--fork.blockNumber`**
+
+=== "Syntax"
+    ```bash
+    --fork.blockNumber=<NUMBER>
+    ```
+
+=== "Example"
+
+    ```bash
+    --fork.blockNumber=182354
+    ```
+
+Block number to fork from. Defaults to the latest block.
+
+**`--fork.preLatestConfirmations`**
+
+=== "Syntax"
+    ```bash
+    --fork.preLatestConfirmations=<NUMBER>
+    ```
+
+=== "Example"
+
+    ```bash
+    --fork.preLatestConfirmations=8
+    ```
+
+When the `--fork.blockNumber` is set to `latest` (default), this option specifies the number of blocks
+before the remote node's latest block to fork from.
+
+The default is `5`.
+
+**`--fork.username`**
+
+=== "Syntax"
+    ```bash
+    --fork.username=<STRING>
+    ```
+
+=== "Example"
+
+    ```bash
+    --fork.username="JohnDoe"
+    ```
+
+Username for basic authentication. Does not require setting `--fork.password`.
+When combined with `--fork.password`, the shorthand is `{ headers: { "Authorization": "Basic {ENCODED-BASIC-HEADER}" } }`
+
+If `--fork.headers` specifies an authorization header, it will be inserted after the Basic token.
+
+**`--fork.password`**
+
+=== "Syntax"
+    ```bash
+    --fork.password=<STRING>
+    ```
+
+=== "Example"
+
+    ```bash
+    --fork.password="changeme"
+    ```
+
+Password for basic authentication. Does not require setting `--fork.username`.
+When combined with `--fork.username`, the shorthand is `{ headers: { "Authorization": "Basic {ENCODED-BASIC-HEADER}" } }`
+
+If the `--fork.headers` specifies an authorization header, it will be inserted after the Basic token.
+
+**`--fork.jwt`**
+
+=== "Syntax"
+    ```bash
+    --fork.jwt=<STRING>
+    ```
+
+=== "Example"
+
+    ```bash
+    --fork.jwt="eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJwZXJtaXNzaW9ucyI6Iio6KiIsInByaXZhY3lQdWJsaWNLZXkiOiIyVUtIM1ZKVGhrT29Lc2tyTEZwd294Q25uUkFSeW9iVjFiRWRnc2VGSFRzPSIsImV4cCI6IjE2MDA4OTk5OTkwMDIiLCJpYXQiOjE2MzkxNTc2Mjd9.FGf-FmfDQlIPCRDGmNnsHZWlwrUr69d7AIDqQrIrUrSJLiwGpR3NCUhVHIDMpQvDHQYf-sFMZTYvZGrvztYRuBKWMbTfIZKN74onzNJbFIPBVQuUX2HMXmI4VQ3UFB11LShiUJHKLna13qdbqfbgJIO3HetxJhJQxTiwtixfHwyPXl-Nx8HbQy_AWH58lLAUeaoLzN7QIA9kborthBpvfK9C7Sv1lXT1cdCDC4oRKBoiMg2RWFZtGtxFsnWyloangwbhCB6Bc_elqY5nd9WkF4ix95xsP_HgBcouy1sDw6jxn5_LveX53H8owczVWP6S1e6hv6hq2fs6YkSntKMK2g"
+    ```
+
+Encoded JSON Web Token (JWT) used for authenticating to servers.
```

Comment:
Worth us adding a link out to https://jwt.io on the `JSON Web Token (JWT)` reference?